### PR TITLE
Be able to refer to pull request from rspec-dev in each rspec gems PR

### PR DESCRIPTION
**Pull request by benoittgt**
I love when we can easily get initial PR when generating PR from rspec-dev. This change add the ability to generate pull request with an additionnal comment.

For example: 
```sh
bundle exec rake "travis:create_pr_with_updates[https://github.com/rspec/rspec-dev/pull/233]"
```

Generates

Capture d’écran 2020-03-30 à 16 32 52

You can also use:
```
$ bundle exec rake 'common_plaintext_files:create_pr_with_updates[Generated from: https://github.com/rspec/rspec-dev/pull/230]'
```

Will generate pull request with body:

```md
These are some updates, generated from rspec-dev's rake tasks.

Generated from: https://github.com/rspec/rspec-dev/pull/230
```

If no extra comment provided, we simply use this body:
```md
These are some updates, generated from rspec-dev's rake tasks.
```

### CODE REVIEW

File: Rakefile [hunk 1]
```diff
@@ -220,11 +220,11 @@ BASE_BRANCH_MAJOR_VERSION = if BASE_BRANCH == 'master'
                               Integer(BASE_BRANCH[/^\d+/])
                             end
 
-def create_pull_request(project_name, branch, base=BASE_BRANCH)
+def create_pull_request(project_name, branch, custom_pr_comment, base=BASE_BRANCH)
   github_client.create_pull_request(
     "rspec/#{project_name}", base, branch,
     "Updates from rspec-dev (#{Date.today.iso8601})",
-    "These are some updates, generated from rspec-dev's rake tasks."
+    "These are some updates, generated from rspec-dev's rake tasks.\n\n#{custom_pr_comment}"
```
Comment (JonRowe) [inline Rakefile]: Should probably check that its present and add the `\n\n` if so:
```suggestion
    ["These are some updates, generated from rspec-dev's rake tasks.", custom_pr_comment].join("\n\n")
```

File: Rakefile [hunk 2]
```diff
@@ -373,7 +373,7 @@ namespace :common_plaintext_files do
 
   desc "Updates the common plaintext files files and creates a PR"
   task :create_pr_with_updates do
-    force_update update_common_plaintext_files_in_repos
+    force_update(update_common_plaintext_files_in_repos, custom_pr_comment)
```
Comment (JonRowe) [inline Rakefile]: Where does `custom_pr_comment` come from?

Comment (benoittgt) [reply Rakefile]: Completely right. https://github.com/rspec/rspec-dev/pull/240/commits/7e6c0f3d8887a4a79a0c3d32184bd598ddb6a55f

**Comment by benoittgt** [comment]
Sorry @pirj I thought it was merged...

@JonRowe are you happy with the last change?

**Comment by pirj** [comment]
Aha!

File: Rakefile [hunk 3]
```diff
@@ -294,8 +294,8 @@ namespace :travis do
   end
 
   desc "Updates the travis files and creates a PR"
-  task :create_pr_with_updates do
-    force_update update_travis_files_in_repos
+  task :create_pr_with_updates, :custom_pr_comment do |t, args|
+    force_update(update_travis_files_in_repos, args[:custom_pr_comment])
```
Comment (JonRowe) [inline Rakefile:298]: Whats this when not provided? `nil` or `""`, as if its `""` it should probably set it to nil.

Comment (benoittgt) [reply Rakefile:303]: With the last commit (https://github.com/rspec/rspec-dev/pull/240/commits/3b1878ca70967dafb7906f5d38b1554996951431) if it's `nil` or an empty string. We do not add the two extra blank line and try to add the custom pull request comment.

I also fixed a missing comma with that task.

Initial PR updated with command result.

**Pull request merged**